src/Discovery/ScheduleDisovery.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Discovery;
+
+use ReflectionClass;
+use ReflectionMethod;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Console\Scheduler\SchedulerConfig;
+use Tempest\Container\Container;
+use Tempest\Discovery\Discovery;
+use Tempest\Support\Reflection\Attributes;
+
+final class ScheduleDisovery implements Discovery
+{
+    private const CACHE_PATH = __DIR__ . '/schedule-discovery.cache.php';
+
+    public function __construct(private SchedulerConfig $schedulerConfig)
+    {
+    }
+
+    public function discover(ReflectionClass $class): void
```

Comment:
Creating a separate discovery for it felt cleaner in some ways but also weird in others - not really sure how to feel about it

Reply:
I'm fine with it, as long as it's tied to a separate config object, it's ok to be separate